File: dotnet/ExtentReports/ExtentReports/Model/MediaList.cs
namespace RelevantCodes.ExtentReports.Model
{
    using System;
    using System.Collections.Generic;

    internal class MediaList
    {
        public List<ScreenCapture> ScreenCapture;
        public List<Screencast> Screencast;

        public MediaList()
        {
            ScreenCapture = new List<ScreenCapture>();
            Screencast = new List<Screencast>();
        }
    }
}

File: java/2.x/src/com/relevantcodes/extentreports/model/MediaList.java
/*
* Copyright (c) 2015, Anshoo Arora (Relevant Codes).  All rights reserved.
* 
* Copyrights licensed under the New BSD License.
* 
* See the accompanying LICENSE file for terms.
*/

package com.relevantcodes.extentreports.model;

import java.util.ArrayList;

public class MediaList {
    public ArrayList<ScreenCapture> screenCapture;
    public ArrayList<Screencast> screencast;
    
    public MediaList() {
        screenCapture = new ArrayList<ScreenCapture>();
        screencast = new ArrayList<Screencast>();
    }
}
